getClients.js

```diff
@@ -0,0 +1,22 @@
+exports.getAll = function(event, context, callback) {
+  const connection = db.connect(event.stageVariables);
+  var query = 'SELECT * FROM Clients ORDER BY' + event.queryStringParameters.sort';
+
+  connection.query(query, function (error, results, fields) {
+    if (error) {
+      db.close(connection);
+      callback(null, {
```

Comment:
AWS Lambda standard is callback(Error error, Object result); In case of an error, it is customary to send the error details in the first argument.. as entire NodeJS is error first callback philosophy.

Sending in result will work but the implementation may not align with the standards